charts/argo-cd/values.yaml

```diff
@@ -279,6 +279,12 @@ configs:
     # -- Enables use of the Progressive Syncs capability
     applicationsetcontroller.enable.progressive.syncs: false
 
+    # When specifying multiple namespaces, specify them in array form
+    applicationsetcontroller.namespaces: []
+    # - namespace1
+    # - namespace2
+    # - namespace3
```

Comment:
All of values under `.Values.configs.params` are set to `argocd-cmd-params-cm` ConfigMap. According to [upstream](https://github.com/argoproj/argo-cd/blob/v2.12.3/docs/operator-manual/argocd-cmd-params-cm.yaml#L205-L206), the type of  `applicationsetcontroller.namespaces` looks string. So I am not sure if it works properly.

Reply:
@yu-croco Thank you for the confirmation! I changed it to a structure that generates ServiceAccount repeatedly for each namespace by entering a comma-separated string instead of a string.